draft-ietf-httpbis-client-cert-field.md

```diff
@@ -241,6 +241,15 @@ Forward proxies and other intermediaries MUST NOT add the `Client-Cert` or
 `Client-Cert` or `Client-Cert-Chain` header field. Similarly, clients MUST NOT
 employ the `Client-Cert` or `Client-Cert-Chain` header field in requests.
 
+When the value of the `Client-Cert` request header is used to select a response
+(e.g., the response content is access-controlled), the response MUST either be
+marked as uncacheable (e.g., `Cache-Control: no-store`) or be designated for
```

Comment:
It doesn't necessarily need to be _marked_  -- some responses just are uncacheable.

```suggestion
uncacheable (e.g., by sending `Cache-Control: no-store`) or be designated for
```